src/plugins/abrt-action-generate-backtrace.c

```diff
@@ -80,6 +80,10 @@ int main(int argc, char **argv)
     struct dump_dir *dd = dd_opendir(dump_dir_name, /*flags:*/ 0);
     if (!dd)
         return 1;
+
+    if (!dd_exist(dd, FILENAME_COREDUMP))
+        abrt_save_coredump(dd, exec_timeout_sec);
```

Comment:
I think it is safer to use { } even for single commands in if-statements. However we do not have coding guidelines for that, so I will leave it to you.